### ISSUE TITLE
Can you provide codes for generating ground truth edge map?

### ISSUE THREAD

**Issue by tyjcbzd** [taozh2017/CFANet#2]
I use inner package in  open-cv to generate ground truth edge map, below is the code:
`edges = cv2.Canny(image, threshold1=100, threshold2=200)`
Do you also use this to generate the edge map, if so, what thresholds did you use, if not, can you pls provide the algorithm? Thanksssss!

**Comment by yike5022** [comment]
Do you apply canny calculations to the original image of the training?

**Comment by taozh2017** [comment]
> Do you apply canny calculations to the original image of the training?

Yes. Edge maps have been put into the training dataset.

**Comment by yike5022** [comment]
Hello, do you also use the Canny algorithm on the polyp dataset? Because the polyp boundary map I generated is very strange.

**Comment by taozh2017** [comment]
Yes. I use the Canny algorithm.

**Comment by yike5022** [comment]
How to choose the parameters of the Canny algorithm? I mean threshold1 and threshold2.

**Comment by yike5022** [comment]
Hello, have you tried to save the boundary map?

**Comment by taozh2017** [comment]
> How to choose the parameters of the Canny algorithm? I mean threshold1 and threshold2.

Do not need to set a threshold, just use Matlab edge detection code. You can email me and we can discuss conveniently.

**Comment by yike5022** [comment]
Hello, what is your email address?

**Comment by IMU-Y** [comment]
Could you share the source code for Canny algorithm to generate edge detection labels ? 
My email : 1124289433@qq.com
Thanks a lot !
